form.js

```diff
@@ -0,0 +1,85 @@
+const Form = document.getElementById('booksform');
+const Bookscontainer = document.querySelector('.books');
+const bookname = Form.title;
+const writer = Form.name;
+
+let title = ''; let
+  namey = '';
+
+const books = [];
+
+function Book(title, name, node = null) {
+  this.id = books.length;
```

Comment:
- [ ] I think it's better to avoid the use of `length` to create `ids`. If you have two items with id 0 and 1. And, delete the item with id 0. After that, create a new item. This new item will receive id 1, the same as the previous one.